packages/app/src/debug/DebugOverLimit.vue

```diff
@@ -27,15 +28,56 @@ import { useI18n } from '@cy/i18n'
 import { getUrlWithParams } from '@packages/frontend-shared/src/utils/getUrlWithParams'
 import { computed } from '@vue/reactivity'
 
+export type DataRetentionLimitExceeded = { __typename: 'DataRetentionLimitExceeded', dataRetentionDays: number | null }
+
+export type UsageLimitExceeded = | { __typename: 'UsageLimitExceeded', monthlyTests: number | null }
+
+export type CloudRunHidingReason = DataRetentionLimitExceeded | UsageLimitExceeded
+
 const { t } = useI18n()
 
 const props = defineProps<{
+  overLimitReason: CloudRunHidingReason | null
   overLimitActionType: OverLimitActionTypeEnum
   overLimitActionUrl: string
+  runAgeDays: number
 }>()
 
 const actionUrl = computed(() => {
   return getUrlWithParams({ url: props.overLimitActionUrl, params: { utmMedium: 'Debug Tab', utmSource: getUtmSource() } })
 })
 
+const isPlanAdmin = computed(() => props.overLimitActionType === 'UPGRADE')
+
+const iconClasses = computed(() => {
+  return [
+    'icon-dark-gray-500',
+    {
+      'icon-dark-secondary-jade-400': !isRetentionLimit(props.overLimitReason),
+      'icon-light-secondary-jade-200': !isRetentionLimit(props.overLimitReason),
+      'icon-dark-secondary-orange-400': isRetentionLimit(props.overLimitReason),
+      'icon-light-secondary-orange-200': isRetentionLimit(props.overLimitReason),
+    },
```

Comment:
```suggestion
   isRetentionLimit(props.overLimitReason)
      ? 'icon-dark-secondary-orange-400 icon-light-secondary-orange-200'
      : 'icon-dark-secondary-jade-400 icon-light-secondary-jade-200',
```

Nit: it's nice to have the classes grouped together